docs/portal/transports/edit-transport.md

```diff
@@ -1,11 +1,11 @@
 # Edit a Transport
 
-To edit a transport, search the transport you want to edit in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Edit".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
+To edit a transport, click the "Action" button of the transport entry in the list and select "Edit". If the transport was not in the current page, you can search it in the search box.
 
 ![null](</docs/resources/images/transports/edit-transport-1.png>)
 
-Edit the items to make the changes you want to make, then click the "Next" button in the top right corner of the page.
+Edit the items to make the changes you want to make, then click the "Next" button in the top-right corner of the page.
```

Comment:
Please consider changing "Edit the items to make the changes you want to make..." to "Edit the items as desired..."
If this string appears elsewhere, please apply this same edit for consistency.